agent/run.sh

```diff
@@ -106,7 +106,7 @@ then
 else
     echo "IMA is not registered yet"
     echo Going to run: node $DIR/main.js --register $BASE_OPTIONS
-    node $DIR/main.js --register $BASE_OPTIONS || true
+    node $DIR/main.js --register3 $BASE_OPTIONS || true
```

Comment:
Did you mean `--register2`????? `--register3` no longer exist

Reply:
yes, it's already updated